docs/source/guide/release_notes/onprem/2.4.3.md

```diff
@@ -0,0 +1,30 @@
+# On-premise Release Notes
+  
+## Label Studio 2.4.3
+
+### Improvement
+- Support simultaneous render of multi-channel audio with added splitchannels="true" option on Audio config (larger memory requirement) [LSDV-3028](https://labelstudio.aha.io/features/LSDV-3028)
+- Allow selecting task automatically on Data Manager page whenever the user presses shift+up/down [LSDV-3093](https://labelstudio.aha.io/features/LSDV-3093)
+- Restore locked annotations [LSDV-3062](https://labelstudio.aha.io/features/LSDV-3062)
+- Navigation back to previous tasks in labeling stream is now persistent against page reloads and user labeling sessions [LSDV-3936](https://labelstudio.aha.io/features/LSDV-3936)
+- Add sync update of is_labeled field [LSDV-2981](https://labelstudio.aha.io/features/LSDV-2981)
+- Improved responsiveness for image regions selection [LSDV-3871](https://labelstudio.aha.io/features/LSDV-3871)
+
+### Customer Bug Fix
+- Fixed bug with presigned TTL setting in cloud storage connection being not persistent [LSDV-2836](https://labelstudio.aha.io/features/LSDV-2836)
+- Slider follows the positional seeker in video when using the step forward or backward buttons. [LSDV-983](https://labelstudio.aha.io/features/LSDV-983)
+- Now it is possible to retrieve the list of uploaded files with api/projects/<project-id>/file-uploads&all=true` request [LSDV-4614](https://labelstudio.aha.io/features/LSDV-4614)
```

Comment:
missing quote

Reply:
https://labelstudio.aha.io/features/LSDV-4614
this is was set like that in RN field

Reply:
fixed both.